maven-surefire-plugin/src/site/apt/featurematrix.apt.vm

```diff
@@ -27,19 +27,19 @@ Feature Matrix
     Not all features are supported for all test frameworks, and the following table gives a brief overview
     of support status:
 
-*---------------------------------------------+-----------+-----------+------------+-----------+----------+
-|| <<Feature>>                                ||<<JUnit3>>||<<JUnit4>>||<<Junit47>>||<<TestNG>>||<<POJO>> |
-*---------------------------------------------+------------+----------+------------+-----------+----------+
-| groups/category support                     |     N      |    N     |      Y     |    Y      |  N       |
-*---------------------------------------------+------------+----------+------------+-----------+----------+
-| security manager support                    |     Y      |    N     |      N     |    N      |  N       |
-*---------------------------------------------+------------+----------+------------+-----------+----------+
-| runOrder support                            |     Y      |    Y     |      Y     |    ?      |  Y       |
-*---------------------------------------------+------------+----------+------------+-----------+----------+
-| run >1 individual test method in a class    |     N      |    Y     |      Y     |    Y      |  N       |
-*---------------------------------------------+------------+----------+------------+-----------+----------+
-| parallel support                            |     N      |    N     |      Y     |    Y      |  N       |
-*---------------------------------------------+------------+----------+------------+-----------+----------+
+*---------------------------------------------+-----------+-----------+------------+-----------+----------+--------------------+
+|| <<Feature>>                                ||<<JUnit3>>||<<JUnit4>>||<<JUnit47>>||<<TestNG>>||<<POJO>> ||<<JUnit Platform>> |
+*---------------------------------------------+------------+----------+------------+-----------+----------+--------------------+
+| groups/category/tags support                |     N      |    N     |      Y     |    Y      |  N       |  Y                 |
+*---------------------------------------------+------------+----------+------------+-----------+----------+--------------------+
+| security manager support                    |     Y      |    N     |      N     |    N      |  N       |  N                 |
+*---------------------------------------------+------------+----------+------------+-----------+----------+--------------------+
+| runOrder support                            |     Y      |    Y     |      Y     |    ?      |  Y       |  N                 |
+*---------------------------------------------+------------+----------+------------+-----------+----------+--------------------+
+| run >1 individual test method in a class    |     N      |    Y     |      Y     |    Y      |  N       |  ?                 |
```

Comment:
Here we have question mark for `run >1 individual test method in a class`. I think this should be `Y`. WDYT?

Reply:
Yes, but there are some corner cases that are not supported, yet (see https://github.com/junit-team/junit5/issues/1343 and https://github.com/junit-team/junit5/issues/1406).

Reply:
Keeping the `?` and adding the links as footnotes to the matrix.